src/main/java/com/google/devtools/build/lib/skyframe/PackageLookupFunction.java

```diff
@@ -330,14 +330,14 @@ private PackageLookupValue computeExternalPackageLookupValue(
     RepositoryValue repositoryValue;
     try {
       repositoryValue = (RepositoryValue) env.getValueOrThrow(
-          repositoryKey, NoSuchPackageException.class, IOException.class, EvalException.class);
+          repositoryKey, NoSuchPackageException.class, IOException.class, EvalException.class, AlreadyReportedException.class);
```

Comment:
is this actually needed? I don't think it's ever thrown, right? Here and below

Reply:
It is, actually. The path is PackageLookupFunction -> RepositoryLoaderFunction -> RepositoryDelegatorFunction. Caught this via a failing test on BazelCI.